openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-board.js

```diff
@@ -362,7 +363,8 @@ var Wb = function() {
 							return false;
 						}
 					}
-				}).resizable({
+				});
+				math.resizable({
```

Comment:
For some reason the resize didn't work if you chain it. It applies the resize in a way so that the window moves while resizing. By not chaining it it works as epxected.